### PR TITLE
Add INSTALL-docker.md [CIRCLE-30366]

### PR DESCRIPTION
**Pull request by lewang**

### CODE REVIEW

File: INSTALL-docker.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Installing the CircleCI Runner on a host with Docker
+
+## Create a Dockerfile that extends the CircleCi Runner image
+
+Here we install python3 on top of the base image
+
+*Dockerfile.runner.extended*
```
Comment (glenjamin) [inline INSTALL-docker.md]:
```suggestion
`Dockerfile.runner.extended`
```

File: INSTALL-docker.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Installing the CircleCI Runner on a host with Docker
```
Comment (glenjamin) [inline INSTALL-docker.md:1]: Perhaps add a bit of preamble here about the execution environment, and how jobs re-use the same container

**Review by glenjamin — APPROVED** [review]

File: INSTALL-docker.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+## Build the Docker image
+
+```bash
+docker build --file ./Dockerfile.runner.extended .
+```
+
+## Start the Docker container
+
+```bash
+CIRCLECI_RESOURCE_CLASS=<resource-class> CIRCLECI_API_TOKEN=<runner-token> docker run --env CIRCLECI_API_TOKEN --env CIRCLECI_RESOURCE_CLASS --name <container-name> <container-id-from-previous-step>
```
Comment (ProjectFrank) [inline INSTALL-docker.md:29]:
```suggestion
docker run --env CIRCLECI_API_TOKEN=<runner-token> --env CIRCLECI_RESOURCE_CLASS=<resource-class> --name <container-name> <image-id-from-previous-step>
```

File: INSTALL-docker.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+# Installing the CircleCI Runner on a host with Docker
+
+## Create a Dockerfile that extends the CircleCi Runner image
```
Comment (ProjectFrank) [inline INSTALL-docker.md]:
```suggestion
## Create a Dockerfile that extends the CircleCI Runner image
```

**Review by ProjectFrank — COMMENTED** [review]
I was able to follow the instructions and run a job. LGTM, pending @glenjamin 's comments

File: INSTALL-docker.md (hunk 3 above)
Comment (lewang) [reply INSTALL-docker.md:29]: @ProjectFrank That's how I originally had it, but the docker command is visible in `docker ps` in some versions of docker client.

**Pull request merged**